k8s/migratev2/scripts/migrate.sh

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+echo "Running datamon migration: v1 => v2"
+echo "==================================="
+echo "datamon repo: ${REPO}"
+echo "datamon source bundle option: ${SOURCEBUNDLE}"
+echo "staging data mounted on: ${STAGING}"
+echo "datamon destination bundle label: ${DESTLABEL}"
+echo "datamon destination context used: ${DESTCONTEXT}"
+echo "Configuration V1:"
+cat ${HOME}/.datamon/datamon.yaml
+
+echo "Configuration V2:"
+cat ${HOME}/.datamon2/datamon.yaml
+
+type datamon1
+type datamon2
+
+# Sanity checks
+if [[ -z ${GOOGLE_APPLICATION_CREDENTIALS} ]] ; then
+  echo "Credentials location is required"
+  exit 1
+fi
+if [[ -z ${REPO} ]] ; then
+  echo "Repo parameter is required"
+  exit 1
+fi
+if [[ -z ${STAGING} ]] ; then
+  echo "staging location parameter is required"
+  exit 1
+fi
+echo "Credentials located at: ${GOOGLE_APPLICATION_CREDENTIALS}"
+if [[ ! -f ${GOOGLE_APPLICATION_CREDENTIALS} ]] ; then
+  echo "Could not find credentials file"
+  exit 1
+fi
+if [[ -z ${SOURCEBUNDLE} ]] ; then
+  echo "Migrating latest bundle"
+fi
+echo "==================================="
+
+res=$(datamon1 repo get --repo ${REPO})
+if [[ $? -ne 0 ]] ; then
+  echo "Failed to get repo on V1 config"
+  exit 1
+fi
+# retrieve the original description of the repo
+description=$(echo $res) # trim space
```

Comment:
this is a neat bash-ism ... if i start using this, however, since the same syntax produces different results in different shells (contrast with `${(z)my_var}` or s'th that exists in zsh only and is an error elsewhere) i'm likely to forget it only works on bash.

this is not a blocker on review, just a note to myself to hopefully remember about this better.